### DISCUSSION TITLE
SemiCircleMarker

### DISCUSSION THREAD

**Discussion by magureal** [python-visualization/folium#2032] [Q&A]
Hi,

What options do I have if I want to create a SemiCircle object (https://python-visualization.github.io/folium/latest/user_guide/plugins/semi_circle.html), with radius in pixels, not meters. Something similar to CircleMarker.

Thanks!

**Comment by hansthen** [comment]
You can find the required calculations here: https://wiki.openstreetmap.org/wiki/Zoom_levels
